test/document_test.rb

```diff
@@ -443,6 +443,17 @@ def setup
     assert_equal [], second_doc.foo
   end
 
+  test 'attachments work properly' do
```

Comment:
`test "Successfully create a dolly document, but not really because I have set up FakeWeb to intercept the response, which leaves me wondering: should I have a full integration test? Anyway, then I will create another FakeWeb response to update the previous dolly document with an attachment. By testing attach_file!, it will test the internals of sending the file to dolly, which is great! But again, should we consider doing the full round trip without FakeWeb so that we know For Sure™ ? Then again, that is probably outside the scope of this PR, so let's worry about that another day" do`